web/assets/main.scss

```diff
@@ -1,10 +1,11 @@
 $govuk-include-default-font-face: false;
 $govuk-moj-table-background-color: #e7f2fb;
-$moj-assets-path: "/supervision/deputy-hub/assets/";
+$moj-assets-path: "/supervision/deputies/public-authority/assets/";
+$govuk-assets-path: "/supervision/deputies/public-authority/assets/";
```

Comment:
Is there a reason why these have the same path? I don't know the difference between these tbh.

Reply:
There is a different asset path variable name for each front end repo. The `$moj-assets-path` variable is for the `@ministryofjustice/frontend `repo and the `$govuk-assets-path` is for the `govuk-frontend` repo. Both need to be set to have the right url to locate the images